webserver/views/api.py

```diff
@@ -140,7 +142,10 @@ def _validate_auth_header():
 
     user = db.user.get_by_token(auth_token)
     if user is None:
-        user = db.user.create_user_with_token(auth_token)
+        if CREATE_FAKE_USERS_FOR_STRESS_TESTING:
```

Comment:
Hmm, I see your point. This is going to be used very rarely, in fact I plan to use it only one more time before our beta. Then this code can go away... But I think adding the guard is a pretty good idea.